tests/lib/python/diffr.py

```diff
@@ -0,0 +1,300 @@
+"""Quick and simple JSON diff util.
+
+Examples:
+    >>> bool(Diff({}, {}))
```

Comment:
I feel that `bool(Diff(A, B))` should return true if A and B are different, rather than the same!

Reply:
This is for consistency with CLI diff commands.